assets/js/base/components/block-error-boundary/block-error.js

```diff
@@ -40,7 +40,7 @@ BlockError.propTypes = {
 	/**
 	 * Error message to display below the content.
 	 */
-	errorMessage: PropTypes.string,
+	errorMessage: PropTypes.oneOfType( [ PropTypes.string, PropTypes.node ] ),
```

Comment:
`PropTypes.node` already includes `PropTypes.string`, so no need to include both.
```suggestion
	errorMessage: PropTypes.node,
```